src/connectors/index.ts

```diff
@@ -2,19 +2,19 @@ import { InjectedConnector } from '@web3-react/injected-connector'
 import { AuthereumConnector } from '@web3-react/authereum-connector'
 import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
 import { NetworkConnector } from './NetworkConnector'
+import { ChainId } from 'dxswap-sdk'
 
 export const INFURA_PROJECT_ID = '0ebf4dd05d6740f482938b8a80860d13'
 
 export const network = new NetworkConnector({
   urls: {
-    1: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
-    4: `https://rinkeby.infura.io/v3/${INFURA_PROJECT_ID}`
+    1: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`
   },
   defaultChainId: 1
 })
 
 export const injected = new InjectedConnector({
-  supportedChainIds: [1, 4, 79377087078960]
+  supportedChainIds: [ChainId.MAINNET, ChainId.RINKEBY, ChainId.ARBITRUM_TESTNET_V3, ChainId.SOKOL]
```

Comment:
No xdai?

Reply:
Will be added as soon as I deploy the contracts there and start the whole integration (so I'd say pretty soon!)